os/se/ameba/rtl_security_api_wrapper.c

```diff
@@ -0,0 +1,408 @@
+/**************************************************************************************************
+*
+* Copyright 2020 Samsung Electronics All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+* either express or implied. See the License for the specific
+* language governing permissions and limitations under the License.
+*
+***************************************************************************************************
+***************************************************************************************************
+*
+*   Copyright (C) 2020 Gregory Nutt. All rights reserved.
+*   Author: Gregory Nutt <gnutt@nuttx.org>
+*
+* Redistribution and use in source and binary forms, with or without
```

Comment:
You don't need to insert Nutt license here

Reply:
Okay, we will remove Nutt from .c file for SE